integration/tests/basic/evm/test_solana_interoperability.py

```diff
@@ -41,6 +43,61 @@ class TestSolanaInteroperability:
     accounts: EthAccounts
     web3_client: NeonChainWeb3Client
 
+    @pytest.fixture(scope="class")
+    def account_sol_network(self, class_account_sol_chain):
```

Comment:
why do you need the fixture which return other fixture without any changes

Reply:
fixed